container_pipeline/lib/command.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+import subprocess
+
+
+def run(cmd):
+    """
+    Runs a shell command.
+
+    :param cmd: Command to run
+    :type cmd: str
+
+    :return: Command output
+    :rtype: str
+    :raises: subprocess.CalledProcessError
+    """
+    return subprocess.check_output(cmd, shell=True)
```

Comment:
Agree with the pipe usage in our code. One such command we're executing is:
```python
# TEST_TAG generation, unique per project                                  
task = subprocess.Popen(                                                
    "date +%s%N | md5sum | base64 | head -c 14",                           
    shell=True,                                                         
    stdout=subprocess.PIPE)                                             
TEST_TAG = task.stdout.read()
```
I'm replacing that `+%s%N` with `+%m%d%Y` so that we can have comparative output.
```python
In [5]: task = subprocess.Popen("date +%m%d%Y | md5sum | base64 | head -c 14", shell=True, stdout=subprocess.PIPE)

In [6]: TEST_TAG = task.stdout.read()

In [7]: TEST_TAG
Out[7]: 'OGQ0NDk5NjVmOD'
```
Without `shell=True`, it gets long and ugly.
```python
In [2]: date_cmd = subprocess.Popen(('date', '+%m%d%Y'), stdout=subprocess.PIPE)

In [3]: md5sum = subprocess.Popen(('md5sum'), stdin=date_cmd.stdout, stdout=subprocess.PIPE)

In [4]: base64_op = subprocess.Popen(('base64'), stdin=md5sum.stdout, stdout=subprocess.PIPE)

In [5]: headc14 = subprocess.check_output(('head', '-c', '14'), stdin=base64_op.stdout)

In [6]: date_cmd.communicate()
Out[6]: ('', None)

In [7]: headc14
Out[7]: 'OGQ0NDk5NjVmOD'
```
The longer approach also calls for some modification in `run_cmd` code, I guess. It should accept `**kwargs`, at least. That enables one to pass `shell=True` if absolutely needed (which can be discussed when someone's going to open such a PR :smile:).

I'm not in favor of or against any approach yet. But, this should help in discussions.